plugins/modules/cloud/misc/terraform.py

```diff
@@ -93,6 +93,13 @@
     type: list
     elements: path
     version_added: '0.2.0'
+  check_destroy:
+    description:
+      - Check and apply only when there are no resource destruction. Note that it only prevents "destroy" actions,
+      but allows "destroy and then create replacement" actions. This option will be ignored if state=absent.
```

Comment:
```suggestion
        but allows "destroy and then create replacement" actions. This option will be ignored if I(state=absent).
```
(The two spaces at the beginning should fix the sanity errors; for the `I(...)` see [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation).)

Reply:
oh thanks, but was it recently added?

Reply:
Do you mean the `I(...)`? It has been around for a long time.